operators/ublhub-operator/2.0.0-beta.4/manifests/ublhub-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,230 @@
+                image: quay.io/projectopenubl/ublhub-operator:v2.0.0-Beta4
+                imagePullPolicy: Always
+                livenessProbe:
+                  failureThreshold: 3
+                  httpGet:
+                    path: /q/health/live
+                    port: 8080
+                    scheme: HTTP
+                  initialDelaySeconds: 0
+                  periodSeconds: 30
+                  successThreshold: 1
+                  timeoutSeconds: 10
+                name: ublhub-operator
+                ports:
+                - containerPort: 8080
+                  name: http
+                  protocol: TCP
+                readinessProbe:
+                  failureThreshold: 3
+                  httpGet:
+                    path: /q/health/ready
+                    port: 8080
+                    scheme: HTTP
+                  initialDelaySeconds: 0
+                  periodSeconds: 30
+                  successThreshold: 1
+                  timeoutSeconds: 10
+                resources:
+                  limits:
+                    cpu: 250m
+                    memory: 256Mi
+                  requests:
+                    cpu: 50m
+                    memory: 64Mi
+              serviceAccountName: ublhub-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ublhub.openubl.io
+          resources:
+          - ublhubs
+          - ublhubs/status
+          - ublhubs/finalizers
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+          - patch
+          - update
+        - apiGroups:
+          - ''
+          resources:
+          - pods
+          - services
+          - services/finalizers
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - '*'
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - '*'
+        - apiGroups:
+          - networking.k8s.io
+          resources:
+          - ingresses
+          - networkpolicies
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          verbs:
+          - '*'
+        serviceAccountName: ublhub-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - sunat
+  - openubl
+  - ublhub
+  links:
+  - name: Website
+    url: https://project-openubl.github.io/
+  - name: Github
+    url: https://github.com/project-openubl/ublhub
+  maintainers:
+  - email: projectopenubl+subscribe@googlegroups.com
+    name: OpenUbl
+  maturity: alpha
+  provider:
+    name: OpenUbl
+  version: 2.0.0-beta.4
```

Comment:
Please add `replaces: ublhub-operator.v2.0.0-beta.3`

Reply:
Thank you very much!